src/Gwen/APIHandlers/SpotifyAPI/load_cookies.py

```diff
@@ -0,0 +1,168 @@
+from selenium.webdriver.common.by import By
+from selenium.common.exceptions import TimeoutException
+
+class Spotify:
+
+
+    #chrome driver
+    webdriver_service = Service(".\chromedriver.exe")
+
+    #options
+    chrome_options=Options()
+    chrome_options.add_argument("--no-sandbox")
+    chrome_options.add_experimental_option("detach", True)
+    #option to remove error: Filed to read descriptor from node connection: A device attached to the system is not functioning
+    chrome_options.add_experimental_option('excludeSwitches', ['enable-logging'])
+
+    #create browser
+    browser = webdriver.Chrome(service=webdriver_service, options=chrome_options)
+
+    #spotipy object API
+    scope = "user-read-playback-state user-modify-playback-state user-read-currently-playing user-read-recently-played streaming user-library-read user-read-private user-library-modify user-read-playback-position playlist-read-private playlist-modify-private"
+    sp = spotipy.Spotify(auth_manager=SpotifyOAuth(client_id=cred.client_id, client_secret=cred.client_secret, redirect_uri=cred.redirect_uri, scope=scope))
+
+    def __init__(self, user, passw):
+        self.user = user
+        self.passw = passw
+
+    def setup(self):
+
+        try:
+            cookies=pickle.load(open('cookies.pkl', 'rb'))
+            return 'cookies already set'
+        except:
+            pass
+        self.chrome_options.add_argument("--headless")
+
+        #open spotify on a new browser
+        self.browser.get(r"https://accounts.spotify.com/en/login?continue=https%3A%2F%2Fopen.spotify.com%2F")
+
+        #wait until page is loaded
+        try:
+            WebDriverWait(self.browser, 20).until(EC.presence_of_element_located((By.XPATH, '//*[@id="login-button"]/div[1]')))
+            #print('page ready')
+        except TimeoutException:
+            print('loading took too much time')
+
+        #insert email
+        email = self.browser.find_element(By.XPATH, '//*[@id="login-username"]')
+        for c in self.user:
+            email.send_keys(c)
+
+        #insert password
+        password = self.browser.find_element(By.XPATH, '//*[@id="login-password"]')
+        for c in self.passw:
+            password.send_keys(c)
+
+        #click login button
+        logbutton = self.browser.find_element(By.XPATH, '//*[@id="login-button"]/div[1]')
+        logbutton.click()
+
+        #wait for page to load
+        try:
+            WebDriverWait(self.browser, 20).until(EC.presence_of_element_located((By.XPATH, '//*[@id="main"]/div/div[2]/div[2]/footer/div/div[2]/div/div[1]/button')))
+            #print('page ready')
+        except TimeoutException:
+            print('loading took too much time')
+        
+        #cookies
+        cookies=self.browser.get_cookies()
+        #print(cookies)
+        pickle.dump(cookies , open("cookies.pkl","wb"))
+
+
+        self.browser.quit()
+
+        return 'cookies have been set'

```

Comment:
Change this to an Enum. Enums are used to pas simple messages such as this

from enum import Enum
class Spotipy:
  class StatusCode(Enum):
    OK = 200
    CREATED = 201
    ACCEPTED = 202
    BAD_REQUEST = 400
    UNAUTHORIZED = 401
    FORBIDDEN = 403
    NOT_FOUND = 404
    SERVER_ERROR = 500

...

def some_function for things(Self):
  ...
  return Spotipy.StatusCode.SERVER_ERROR

Lmk if this doesn't make sense